l10n_it_fatturapa/bindings/_ds.py

```diff
@@ -1,35 +1,23 @@
 # flake8: noqa
 # -*- coding: utf-8 -*-
 # PyXB bindings for NM:f1c343a882e7a65fb879f4ee813309f8231f28c8
-# Generated 2016-12-12 16:50:59.495293
-# by PyXB version 1.2.5 using Python 2.7.12.final.0
-# By Lorenzo Battistini <lorenzo.battistini@agilebg.com>
+# Generated 2019-03-18 02:57:28.772754 by PyXB version 1.2.6 using Python 2.7.15.final.0
 # Namespace http://www.w3.org/2000/09/xmldsig# [xmlns:ds]
 
 from __future__ import unicode_literals
-import logging
+import pyxb
+import pyxb.binding
+import pyxb.binding.saxer
 import io
-
-_logger = logging.getLogger(__name__)
-
-try:
```

Comment:
il travis si lamenta della rimozione di questo try, per quale motivo era stato tolto? scusate se non l'ho visto magari nella conversazione, non lo trovo

Reply:
> il travis si lamenta della rimozione di questo try

In quale riga di https://travis-ci.org/OCA/l10n-italy/jobs/507972110 vedi questo?

Reply:
https://travis-ci.org/OCA/l10n-italy/jobs/507972110#L633 in avanti, sbaglio?

Reply:
scusa, L642

Reply:
ho spostato il check e il log in `__init__.py`

Reply:
@sergiocorato ah ok, sì, però come anche tutti gli altri import di `_ds.py`. Quindi direi che ce ne possiamo disinteressare